ethcore/src/block.rs

```diff
@@ -498,7 +498,7 @@ pub fn enact(
 	tracing: bool,
 	db: Box<JournalDB>,
 	parent: &Header,
-	last_hashes: LastHashes,
+	last_hashes: Arc<LastHashes>,
```

Comment:
and then since `LastHashes` appears to be just a `Vec<H256>`, why not just have the last hashes field be a `&[H256]`?